linkcovery/core/utils.py

```
@@ -83,35 +85,31 @@ def normalize_url(url: str) -> str:
+            attrs = dict(attrs)
+
+            # match: <meta name="description" content="...">
+            if attrs.get("name", "").lower() == "description":
+                self.description = attrs.get("content", "").strip()
+
+
+async def fetch_description(url: str) -> str:
+    async with AsyncClient(
+        timeout=10,
+        follow_redirects=True,
+        verify=False,
```

Ruff (0.14.4)

Probable use of `httpx` call with `verify=False` disabling SSL certificate checks

(S501)